charts/metallb/templates/speaker.yaml

```diff
@@ -107,7 +107,7 @@ data:
   frr.conf: |+
     ! This file gets overriden the first time the speaker renders a config.
     ! So anything configured here is only temporary.
-    frr version 7.5.1
+    frr version 8.0
```

Comment:
because that refers to the format of the file. What we care here is to have it 8.0+ compatible, so whoever wants to use an older frr version (for stability reasons for example) won't hit any problem.

Reply:
Do we state somewhere we "have 8.0+ compatible frr config"?

Reply:
nope and we shouldn't care much. 7.5 is not supported anyway